CefSharp.Core/Internals/CefBrowserHostWrapper.cpp

```diff
@@ -1,4 +1,4 @@
-// Copyright � 2010-2017 The CefSharp Authors. All rights reserved.
+// Copyright ?2010-2017 The CefSharp Authors. All rights reserved.
```

Comment:
This and other similar changes should be reverted - the copyright sign should remain in the previous encoding. (UTF-8 I presume)